x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/kibana/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,224 @@
+
+export function getKibanaInstructionsForEnablingMetricbeat(product, _meta, {
+  esMonitoringUrl,
+  hasCheckedStatus,
+  checkingMigrationStatus,
+  checkForMigrationStatus,
+  autoCheckIntervalInMs
+}) {
+  const installMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.installMetricbeatTitle', {
+      defaultMessage: 'Install Metricbeat on the same server as Kibana'
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.kibanaInstructions.installMetricbeatDescription"
+            defaultMessage="Follow {link}."
+            values={{
+              link: (
+                <EuiLink href="https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-installation.html" target="_blank">
+                  <FormattedMessage
+                    id="xpack.monitoring.metricbeatMigration.kibanaInstructions.installMetricbeatLinkText"
+                    defaultMessage="the instructions here"
+                  />
+                </EuiLink>
+              )
+            }}
+          />
+        </p>
+      </EuiText>
+    )
+  };
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the Kibana module in Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+          metricbeat modules enable kibana-xpack
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.kibanaInstructions.enableMetricbeatModuleDescription"
+              defaultMessage="By default the module will collect Kibana monitoring metrics from http://localhost:5601. If the local Kibana instance has a different address, you must specify it via the {hosts} setting in the {file} file."
+              values={{
+                hosts: (
+                  <Monospace>hosts</Monospace>
+                ),
+                file: (
+                  <Monospace>modules.d/kibana-xpack.yml</Monospace>
+                )
+              }}
+            />
+          </p>
+        </EuiText>
+      </Fragment>
+    )
+  };
+
+  const configureMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.configureMetricbeatTitle', {
+      defaultMessage: 'Configure Metricbeat to send to the monitoring cluster'
+    }),
+    children: (
+      <Fragment>
+        <EuiText>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.kibanaInstructions.configureMetricbeatDescription"
+            defaultMessage="Make these changes in your {file}."
+            values={{
+              file: (
+                <Monospace>metricbeat.yml</Monospace>
```

Comment:
Per https://www.elastic.co/guide/en/kibana/master/monitoring-metricbeat.html (step 7), there is extra stuff you need to do if security is enabled on the monitoring cluster.